src/components/POLine/Eresources/EresourcesForm.js

```diff
@@ -33,10 +33,11 @@ class EresourcesForm extends Component {
     const { parentResources, order } = this.props;
     const created = get(order, 'metadata.createdDate', '');
     const vendors = get(parentResources, 'vendors.records', []);
-    const vendorOptions = vendors.map((v) => ({
-      label: v.name,
-      value: v.id,
-    }));
+    const vendorOptions = vendors.sort((a, b) => a.name.localeCompare(b.name))
```

Comment:
What do you think about URL param? It looks to me like back-end work - to return sorted items

Reply:
Sure, it's better. Thanks. I added params to GET vendors